vtex/faststore
# feat: Move ProductTitles section to app folder

## What's the purpose of this pull request?

Move ProductTitles section to app folder

## How it works?

![chrome-capture-2024-8-19](https://github.com/user-attachments/assets/9f9f29a8-bf2b-4e7d-a12d-3d3757ed941b)


### How to test it?

```
Run yarn dev
Make sure that all the current pages are working as expected (/, /office and some /<slug>/p);
Make sure /fs-next-update is rendering with the ProductTitles section without errors.
```
